tests/cases/data/ConnectionsTest.php

```diff
@@ -76,7 +76,7 @@ public function testConnectionGetAndReset() {
 		$msg = "Cannot connect to localhost:{$this->_port}";
 		$this->skipIf(!$this->_canConnect('localhost', $this->_port), $msg);
 
-		$expected = $this->config + ['type' => 'database', 'filters' => []];
+		$expected = $this->config + ['type' => 'database'];
```

Comment:
Please Re-Rebase the mongo branch onto the current `2.0` I've cherry picked that commit there, as it solve a problem on that branch.